MerlinAU.sh

```diff
@@ -4739,7 +4746,7 @@ Please manually update to version $minimum_supported_version or higher to use th
            then
                # Diversion unmount command also unloads entware services #
                Say "Stopping Diversion service..."
-               /opt/bin/diversion unmount >/dev/null
+               /opt/bin/diversion unmount &
```

Comment:
Why are we running this in the background again?
Shouldn't we wait for diversion to unmount before we start flashing?

Reply:
Oh wait. Nvm I went back through the PRs and it was cause Gnuton was crashing and freezing when trying to unmount.

Yes yes. My memory is coming back to me now hahah

Reply:
2 nights in a row of little sleep is starting to showwww.

Can we still hide the output though? So we don't see the diversion splash right before the flash.

Reply:
> Why are we running this in the background again? Shouldn't we wait for diversion to unmount before we start flashing?
>
If Diversion has not completed & returned after 5 seconds, something has gone wrong, at which point we don't want to hang the F/W flash process because of it. We just continue to eject the USB drive and move on. Remember that by this point we've already stopped all other Entware services except for Diversion & Tailscale (if enabled).

Reply:
> 2 nights in a row of little sleep is starting to showwww.
> 
> Can we still hide the output though? So we don't see the diversion splash right before the flash.
>
I thought it was good for the user to see that Diversion has been unmounted when running interactively.
But either way is fine with me. I have no preference.